### PR TITLE
Fix wrong link and typo

### PR DESCRIPTION
**Pull request by Davidffry**
**What this PR does / why we need it**: Change / Fix / Add feature for documentation

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: No Issue fix
Fixes #

**Special notes for your reviewer**: I think we need to change all the **oc command** in the documentation, to be more agnostic. If some specificities in Openshift, either create each time if necessary a dedicated chapter or push it outside the kubevirt.
Because you ask to try with the quickstart **but** we can't use the **oc command**.  

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [x] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [x] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [x] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jean-edouard** [comment]
Thanks @Davidffry
PR makes sense and special notes too. Any reason to keep this as a draft?
/approve

**Comment by Davidffry** [comment]
> Thanks @Davidffry PR makes sense and special notes too. Any reason to keep this as a draft?

No really reason, but re-read I view, in the header, that there is a note with this _"By deploying KubeVirt on top of OpenShift the user can benefit from the [OpenShift Template](https://docs.openshift.com/container-platform/4.10/virt/vm_templates/virt-creating-vm-template.html) functionality."_ 

There is no ***Template*** CRD in Kubevirt/Kubernetes deployment.

File: docs/virtual_machines/templates.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ the kubevirt services.
 
 Here's a description of the kubevirt annotations. Unless otherwise
 specified, the following keys are meant to be top-level entries of the
-template metadata, like
+template metadata, like (for Openshift console to display)
```
Comment (aburdenthehand) [inline docs/virtual_machines/templates.md:240]:
```suggestion
template metadata. For OpenShift console to display:
```

**Review by aburdenthehand — COMMENTED** [review]
Thanks for the PR @Davidffry !
I have one suggestion to make it a bit easier to parse as having a parenthesis after the "like" makes it seems like the sentence has been cut short. 
I see other annotation examples still have it (which we should change in a subsequent PR), but hopefully this makes this one a bit clearer.

**Comment by fabiand** [comment]
@Davidffry Thanks for the contribution!

Can you rebase and address Andrew's comment?

Thanks!

/lgtm

**Comment by fabiand** [comment]
/remove-lgtm

Andrew, please lgtm once we are good

**Pull request closed without merging**